plugin-packs/postcss-preset-env/test/basic.css

```diff
@@ -354,7 +354,7 @@
 	right: round(nearest, 2.5px, 1px);
 	bottom: round(up, 2.5px, 1px);
 	left: round(down, 2.5px, 1px);
-	inset: round(to-zero, 2.5px, 1px);
```

Comment:
This test is for `round()` but logical transforms `inset`.
This confused me so I changed the test to `padding-left`